### DISCUSSION TITLE
Clarify license

### DISCUSSION THREAD

**Discussion by MariusVolkhart** [sualeh/creditcardnumber#8] [General]
The POM indicates that this project is licensed under Eclipse license. The `LICENSE` file in the `.github` folder indicates LGPL. Could you please clarify which license applies? Is the project dual-licensed?

**Comment by sualeh** [comment]
@MariusVolkhart - this project is licensed under either LGPL or the Eclipse license. If you download and use the code, you can choose to follow the terms of either one of those two licenses. If you need to use this software under some other terms, I can work with you on commercial licensing terms, if you contact me at sualeh@hotmail.com.

**Comment by sualeh** [comment]
@MariusVolkhart - thank you for your contribution to creditcardnumber. Please have your team sign the Contributor Agreement. Please read ".github/CLA.md" carefully, and put signatory names in the table at the bottom. Please commit this change as a verified commit, and submit a pull request. Thanks!
